phpBB/styles/prosilver/template/mcp_post.html

```diff
@@ -168,6 +168,10 @@
 			</form>
 		<!-- ENDIF -->
 
+		<!-- IF S_MCP_POST_ADDITIONAL_OPTS -->
```

Comment:
I think the event should be used without the `IF`.
Extensions can check for that themselves and not having it allows extensions which only display when no additional options are displayed.